notebook/converted/benchmark_results.py

```diff
@@ -1485,7 +1485,7 @@ def return_unlabeled():
 tissue_metrics
 
 
-# In[4]:
+# In[9]:
```

Comment:
Do you want to plot the results with and without scRNA-seq samples removed here, instead of overwriting the old ones? Or are you sure you'll only use the dataset with scRNA-seq samples removed going forward?

Reply:
I plan on only using the dataset without scRNA-seq samples going forward.